AZ3166/tests/UnitTest/MemoryStatusTest.ino

```diff
@@ -0,0 +1,49 @@
+test(memory_heap)
```

Comment:
consider to merge into one test call

Reply:
one test() method means one test scenario, and all test() methods belong to one sketch file that seems there is no necessary to merge into one test()